locale/cups_it.po

```diff
@@ -15326,6 +15560,135 @@ msgstr ""
 
 msgid "~/.cups/lpoptions file names default destination that does not exist."
 msgstr ""
+"Il file ~/.cups/lpoptions fa riferimento a una destinazione predefinita che "
+"non esiste."
+
+#, c-format
```

Comment:
The new strings from here to the end look like from master branch, please remove them since the PR is for 2.4.x branch.